README.md

```diff
@@ -32,6 +32,38 @@ Setup
 Changelog
 ---------
 
+### 0.4
```

Comment:
With https://github.com/alleyinteractive/searchpress/pull/125 being included in 0.4 would it be worthwhile to call that out as a breaking change to indexing?

Reply:
🤦‍♂️ yes, completely forgot